WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -669,6 +685,42 @@ extension ProductsViewController: SyncingCoordinatorDelegate {
 
         ServiceLocator.stores.dispatch(action)
     }
+
+    /// Update local Products Settings (eg. sort order or filters stored in Products settings)
+    ///
+    func updateLocalProductSettings(sort: ProductsSortOrder? = nil,
+                                    stockStatusFilter: ProductStockStatus? = nil,
+                                    productStatusFilter: ProductStatus? = nil,
+                                    productTypeFilter: ProductType? = nil) {
```

Comment:
ℹ️ This can be `private`. 🙂 

```suggestion
    private func updateLocalProductSettings(sort: ProductsSortOrder? = nil,
                                            stockStatusFilter: ProductStockStatus? = nil,
                                            productStatusFilter: ProductStatus? = nil,
                                            productTypeFilter: ProductType? = nil) {
```